src/components/quick-links/quick-links.stories.js

```diff
@@ -0,0 +1,56 @@
+import { html, repeat } from ".storybook/helpers";
+import { withKnobs, number, text } from "@storybook/addon-knobs";
+import image4x3 from ".storybook/assets/images/greek-hall/4x3/1240.jpg";
+
+
+export default {
+	title: "Quick Links",
+	decorators: [withKnobs],
```

Comment:
would prefer migrating to controls/args instead of knobs (see https://github.com/metmuseum/marble/issues/132)